src/gui/Src/Utils/Configuration.cpp

```diff
@@ -529,16 +526,10 @@ const bool Configuration::getBool(const QString category, const QString id) cons
     {
         if(Bools[category].contains(id))
             return Bools[category][id];
-        QMessageBox msg(QMessageBox::Warning, "NOT FOUND IN CONFIG!", category + ":" + id);
-        msg.setWindowIcon(QIcon(":/icons/images/compile-warning.png"));
-        msg.setWindowFlags(msg.windowFlags() & (~Qt::WindowContextHelpButtonHint));
-        msg.exec();
+        GuiAddLogMessage(QString("NOT FOUND IN CONFIG! %1:%2").arg(category).arg(id).toUtf8().constData());
         return false;
     }
-    QMessageBox msg(QMessageBox::Warning, "NOT FOUND IN CONFIG!", category);
-    msg.setWindowIcon(QIcon(":/icons/images/compile-warning.png"));
-    msg.setWindowFlags(msg.windowFlags() & (~Qt::WindowContextHelpButtonHint));
-    msg.exec();
+    GuiAddLogMessage(QString("NOT FOUND IN CONFIG! %1").arg(category).toUtf8().constData());

```

Comment:
this will crash when an actual config string has not been found before the MainWindow is loaded (this was the reason these QMessageBoxes are here)